beakerx/beakerx/environment.py

```diff
@@ -20,6 +20,7 @@
 default_config = """
 {
     "beakerx": {
+        "version": 1.1
```

Comment:
here the version is floating point but below it is a string.
IMO integer is best.